creme/geolocation/locale/fr/LC_MESSAGES/django.po

```diff
@@ -186,6 +186,9 @@ msgstr "autour de"
 msgid "No geolocated address for now"
 msgstr "Aucune adresse géolocalisée pour le moment"
 
+msgid "The geolocation feature is disabled for entities in the trash"
+msgstr "La géolocalisation est désactivée pour les fiches dans la corbeille"
+
```

Comment:
The date in the header should be set (use the command 'makemessages', it updates the header, avoids typo in the message key, sets the messages in the right order etc...).